appsscript.json

```diff
@@ -1,10 +1,10 @@
 {
-  "timeZone": "America/New_York",
+  "timeZone": "America/Los_Angeles",
   "dependencies": {},
   "exceptionLogging": "STACKDRIVER",
   "runtimeVersion": "V8",
   "webapp": {
     "executeAs": "USER_DEPLOYING",
     "access": "MYSELF"
   }
-}
+}
```

Comment:
nit: newline

Reply:
Not 100% clear on the issue here with the newline?